src/jit/lower.cpp

```diff
@@ -1546,7 +1546,7 @@ void Lowering::LowerCall(GenTree* node)
             }
 #endif
 #if defined(_TARGET_AMD64_) && defined(FEATURE_READYTORUN_COMPILER)
-            if (call->gtEntryPoint.addr != nullptr)
+            if ((call->gtEntryPoint.addr != nullptr) && !call->IsFastTailCall())
```

Comment:
Please add a comment, why we don't let non-virt calls to be lowered if it is not a fast tail call (e.g. tail call via helper).  just for my education, does ready-to-run support helper based tail calls?